### PR TITLE
add marker types -- new icons are placeholders

### PR DESCRIPTION
**Pull request by cloudkucooland**
I know sned is working on the linters; but I'd like this to go in so I can work on the server-side parts for markers. Please review and approve quickly.

### CODE REVIEW

File: src/code/mapDrawing.js [hunk 1]
```diff
@@ -93,15 +93,28 @@ export const getPopupBodyWithType = (portal, target) => {
 
 //** This function returns the appropriate image for a marker type */
 const getImageFromMarkerType = (type) => {
+    /* SCB: why not just cram these in a map type */
```
Comment (TheSned) [inline src/code/mapDrawing.js]: We could do that with the implementation of the function, but I still like it as a function. You also still have to handle the `default:` case.

File: src/code/scopes.js [hunk 2]
```diff
@@ -34,33 +34,42 @@
...
             markerIcon: scope.static.images.marker_alert_decay
-        }
-        /*, {
+        }, {
+	    name: Wasabee.Constants.MARKER_TYPE_KEY,
+	    label: "get key",
+	    color: "#7D7D7D",
+	    markerIcon: scope.static.images.marker_alert_key
+	}, {
+	    name: Wasabee.Constants.MARKER_TYPE_LINK,
```
Comment (TheSned) [inline src/code/scopes.js]: It looks like this file somehow got messed up. I see a lot of places where name is repeated.

File: src/code/mapDrawing.js (hunk 1 above)
Comment (cloudkucooland) [reply src/code/mapDrawing.js]: if you like it as a function, leave it be.

**Comment by cloudkucooland** [comment]
hrm. not sure what's going on there.

**Pull request closed without merging**